lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyHoverParticipant.java

```diff
@@ -156,6 +161,7 @@ private Hover getHoverFeatureDescription(String featureName, DOMDocument documen
 
         // if this is a versionless feature, do not add any info about enables or enabled by
         if (flNode.isVersionless()) {
+            addFeatureListSource(sb);
```

Comment:
The source line should go after the `.` that is getting added for the description in the next 3 lines of code.